Geocoding.md

```diff
@@ -5,25 +5,25 @@
 #Contents
 
 - [Description](#description)
-- [Request Structure](#request-structure)
+- [Geocoding Request](#geocoding-request)
+- [Geocoding Request Structure](#geocoding-request-structure)
+- [Place ID Lookup Request](#place-id-lookup-request)
+- [Place ID Lookup Request Structure](#place-id-lookup-request-structure)
 - [Response Structure](#response-structure)
 - [Versioning](#versioning)
 
 ----------------------------
 #Description
 
-Accepts structured or unstructured location information and attempts to resolve the input to a geographic coordinate. If successful, the response will contain one (or more, if requested) locations with latitude and longitude information as well as any available administrative location information, such as country and state. This service relies on Google's Geocoding API as its data vendor; Google's service is documented [here](https://developers.google.com/maps/documentation/geocoding/intro). This service supports the HTTP/GET method only.
-
-#Place Id Requests
-
-The service also accepts Google Place Id's as a means to retrieve goeographic information. Requesting a location using a Google Place Id requires that a request only contains the Place Id and a culture param if language specification is desired. Additional location parameters will result in a 400 response in this request.
+This service makes use of two different requests: Geocoding Requests and Google Place ID Lookup. It accepts structured or unstructured location information and attempts to resolve the input to a geographic coordinate. If successful, the response will contain one (or more, if requested) locations with latitude and longitude information as well as any available administrative location information, such as country and state. This service relies on Google's Geocoding API as its data vendor; Google's service is documented [here](https://developers.google.com/maps/documentation/geocoding/intro). This service supports the HTTP/GET method only.
```

Comment:
The second and third sentences in this paragraph have fallen out of date with the intro of place_id lookups. (The second is obvious; the third is because place_id only ever returns one result.) I would replace those sentences with something like: "The two request types represent two different methods of obtaining geographic information for a query (latitude/longitude, locality, administrative areas, country, etc.) and are explained in more detail below."

Oh, and ever since geo was blankspaced, the very last sentence no longer holds true. Could you change that to something like: "This service supports both HTTP GET and POST methods. Requests will be handled the same way regardless of HTTP method; use whichever method you find most convenient."

(I add that second sentence here because POST typically implies different behavior than GET. Our services typically respond to both methods in the same way, which is a deviation from standard RESTful API standards.)